EnableJupyterDisplay.swift

```diff
@@ -0,0 +1,178 @@
+#if canImport(Cryptor)
```

Comment:
A license header is necessary here. Could you copy and past it from one of the existing files?

Reply:
@rxwei Should I change the year to 2019, in the header that I add?

Reply:
Sounds good!